internal/clients/stubs/ec2_stub.go

```diff
@@ -14,7 +15,8 @@ type ec2CtxKeyType string
 var ec2CtxKey ec2CtxKeyType = "ec2-interface"
 
 type EC2ClientStub struct {
-	Imported []*models.Pubkey
+	Imported []*types.KeyPairInfo
+	lastId   int
```

Comment:
Heh, what this is used for? Why don't you simply use `len(Imported)`?

Reply:
because for some reason it didn't occure to me xD Good idea :+1: